plugins/GSdx/res/tfx.fx

```diff
@@ -516,10 +517,15 @@ float4 tfx(float4 t, float4 c)
 void datst(PS_INPUT input)
 {
 #if PS_DATE > 0
-	float alpha = sample_rt(input.tp.xy).a;
 #if SHADER_MODEL >= 0x400
+#if PS_WRITE_RG == 1
```

Comment:
PS_WRITE_RG check doesn't need to fall under SHADER_MODEL since we are using `#if`, the code can be reduced.